manifests/chrome.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.0",
+  "version": "0.0.1",
```

Comment:
This is overwritten by the release script, so it should have a fixed value (just for the development)